test/data/vm-data.json

```diff
@@ -0,0 +1,159 @@
+    "coinbase": "0x8888f1f195afa192cfee860698584c030f4c9db1",
+    "difficulty": "0x020000",
+    "extraData": "0x42",
+    "gasLimit": "0x023e38",
+    "gasUsed": "0x00",
+    "hash": "0xce1f26f798dd03c8782d63b3e42e79a64eaea5694ea686ac5d7ce3df5171d1ae",
+    "mixHash": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+    "nonce": "0x0102030405060708",
+    "number": "0x00",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "receiptTrie": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+    "stateRoot": "0xaf81e09f8c46ca322193edfda764fa7e88e81923f802f1d325ec0b0308ac2cd0",
+    "timestamp": "0x54c98c81",
+    "transactionsTrie": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+    "uncleHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"
+  },
+  "genesisRLP": "0xf901fcf901f7a00000000000000000000000000000000000000000000000000000000000000000a01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347948888f1f195afa192cfee860698584c030f4c9db1a0af81e09f8c46ca322193edfda764fa7e88e81923f802f1d325ec0b0308ac2cd0a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421b9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000830200008083023e38808454c98c8142a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421880102030405060708c0c0",
+  "lastblockhash": "0xf53f268d23a71e85c7d6d83a9504298712b84c1a2ba220441c86eeda0bf0b6e3",
+  "postState": {
+    "8888f1f195afa192cfee860698584c030f4c9db1": {
+      "balance": "0x456391824508d41c",
+      "code": "0x",
+      "nonce": "0x00",
+      "storage": {}
+    },
+    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+      "balance": "0x4a723dc6b40b8cedf70fa8",
+      "code": "0x",
+      "nonce": "0x07",
+      "storage": {}
+    },
+    "bbbf5374fce5edbc8e2a8697c15331677e6ebf0b": {
+      "balance": "0x3c",
+      "code": "0x",
+      "nonce": "0x00",
+      "storage": {}
+    }
+  },
```

Comment:
note to self: This is the final state of the accounts. One of the operations creates a new account.